### DISCUSSION TITLE
WECC (xlsx) System Architecture

### DISCUSSION THREAD

**Discussion by TimThacker** [CURENT/andes#405] [Modeling] [answered]
Greetings Hantao,

I found in the ANDES documentation that Professor Kai Sun's 179-Bus WECC System is credited for the WECC case, however I believethat the system architecture in the .xlsx WECC system did not match the system diagram here: http://web.eecs.utk.edu/~kaisun/Oscillation/basecase.html

It should be noted that Prof. Kai Sun's base case is a simpler system and has a number of differences such as models used. I am curious about the system architecture, however. Do you know where I could find a diagram that reflects the ANDES 179-Bus WECC System? Is the diagram at the link a proper representation? Thank you.

**Comment by cuihantao** [comment, marked as answer]
I remember they are the same system except for the numbering. But indeed there are many versions of the system, and I couldn’t recall the details off my head. 

The last two buses in the 181-bus system, compared with the 179-bus case, are the only buses added. Two synchronous generators are added to the two new buses.
